hhpc.c

```diff
@@ -236,7 +237,18 @@ static int parseOptions(int argc, char *argv[]) {
         }
     }
 
-    return 1;
+    if (optind == argc) {
+        return 1;
+    }
+    else {
+        printf("%s: wrong number of arguments -- expected none, found:", argv[0]);
```

Comment:
This should go to stderr IMHO (there are other `fprintf(stderr,` calls in the codebase.

Reply:
Yeah, that occurred to me afterwards. Fixed.